src/main.js

```diff
@@ -137,10 +137,15 @@ export const enableGeoTIFFTileSource = (OpenSeadragon) => {
           // find unique aspect ratios (with tolerance to account for rounding)
           const tolerance = 0.015;
 
+          // Organize images into sets based on aspect ratio as well as
+          // macro thumbnails and labels according to the Aperio SVS format:
+          //   https://web.archive.org/web/20120420105738/http://www.aperio.com/documents/api/Aperio_Digital_Slides_and_Third-party_data_interchange.pdf (pg 14)
           const aspectRatioSets = images.reduce((accumulator, image) => {
             const r = image.getWidth() / image.getHeight();
+            const s = image.fileDirectory.ImageDescription.split("\n")[1];
```

Comment:
Is `ImageDescription` guaranteed to exist as a field within `fileDirectory` for all TIFF files? If not (or if it is missing even if against spec) this line will break the plugin. I think it would be smart to do something like:

```
if(image.fileDirectory.ImageDescription){
...
}
```

Reply:
That is a fair check, I'll add this.  I am certain there are remote possibilities of edge cases where it is missing.